harness/tests/run_godot_kotlin_tests.sh

```diff
@@ -0,0 +1 @@
+$1 -d -s --path $PWD addons/gut/gut_cmdln.gd | ( no_error="false"; jvm_closed="false"; while read -r line
do
    echo "$line"
    if echo "$line" | grep -q "Failing asserts:  0"; then
        no_error="true"
    fi
    if echo "$line" | grep -q "Shutting down JVM ..."; then
        jvm_closed="true"
    fi
done
if [[ "$no_error" == "false" ]]; then
    echo "ERROR: Some assertions failed !"
    exit 1
elif [[ "$jvm_closed" == "false" ]]; then
    echo "ERROR: JVM has not closed properly !"
    exit 1
else
    echo "ERROR: Some assertions failed !"
    exit 0
fi
); exit $?
```

Comment:
Hmm somethings off here... We have many printed carriage returns in this script ( `^M`) . Did you copy it from windows or something?
Maybe this helps?: https://stackoverflow.com/questions/15020883/unix-script-appends-m-at-end-of-each-line

Reply:
yep ! that was the problem :p